pipenv/cli.py

```diff
@@ -209,7 +209,7 @@ def do_install_dependencies(dev=False, only=False, bare=False, requirements=Fals
     else:
         if not bare:
             click.echo(crayons.yellow('Installing dependencies from Pipfile.lock...'))
-        with open(project.lockfile_location, 'r') as f:
+        with open(project.lockfile_location) as f:
```

Comment:
@nateprewitt I think I found why it failed. The reason is that I removed `'r'` argument from `codecs.open` at line `540` of `cli.py`